compiler/noirc_frontend/src/hir/comptime/tests.rs

```diff
@@ -19,6 +19,8 @@ use crate::node_interner::FuncId;
 use crate::parse_program;
 
 /// Create an interpreter for a code snippet and pass it to a test function.
+///
+/// The `stdlib` is not made available as a dependency.
```

Comment:
```suggestion
/// The stdlib is not made available as a dependency.
```
Since stdlib isn't a name in noir code